Formalizing a process for joining customer calls (#1615)
sourcegraph/about: handbook/sales/onboarding/joining_customer_calls.md

> # Joining customer calls

Reviewer: It's annoying to go make this change everywhere, but could we at least change them on this page to "customer and prospect" calls? Important distinction to use internally.

Author: done (everywhere on this page)!

Joining customer calls is a valuable part of any new team member's onboarding or ongoing work. It helps you quickly learn what customers are directly interested in, and how we communicate the product to them.

## Types of calls

While calendar events won't have one of these exact lables, knowing the types of calls will help you categorize calls so you can join the ones you'll find most helpful. 

> 1. **New prospect:** the first call with a prospect, who may not know much about Sourcegraph. Often named "Intro Call."

Reviewer: ```suggestion
1. **New prospect:** the first call with a prospect, who may not know much about Sourcegraph. Often named "Intro Call." These can be useful for understanding how we communicate the value of Sourcegraph to buyers and admins.
```

> 1. **Installation:** to walk through setting up Sourcegraph with a technical point of contact.  

Reviewer: ```suggestion
1. **Installations or technical check-ins:** to walk through setting up Sourcegraph with a technical point of contact. These can be useful for learning what types of technical blockers users and admins encounter, and how we help solve them.
```

> 1. **Trial check-in:** to discuss how things are going during a trial.

Reviewer: ```suggestion
1. **Trial check-in:** to discuss how things are going during a trial. These are often recurring weekly, and can be quite informal. These can be useful for getting a snapshot of a deal process, or to hear mid-flight product or process feedback.
```

> 1. **Training session:** a demo walkthrough of common use cases, an advanced tutorial, or a Q&A, all a customer's end users. Often have names like "Sourcegraph 101," "Webinar,"" "Demo," "Q&A.""

Reviewer: ```suggestion
1. **Training session:** a demo walkthrough of common use cases, an advanced tutorial, or a Q&A, all a customer's end users. Often have names like "Sourcegraph 101," "Webinar,"" "Demo," "Q&A." These can be useful for understanding how we help new end users understand Sourcegraph's value, and what sorts of questions new users ask."
```

> 1. **Existing customer check-in:** to hear how things might be improved or learn about exciting ways a customer is using Sourcegraph. 

Reviewer: ```suggestion
1. **Existing customer check-in:** to hear how things might be improved or learn about exciting ways a customer is using Sourcegraph. These can vary significantly, so it can be worthwhile to check in with the team to understand context in advance.
```

## How to join calls to listen

> To join in on calls, you should first see which calls are happening that week and then slack message the event organizer to ask if you can join. Look at Sales or CE team members' calendars to find calls. 

Reviewer: ```suggestion
To join in on calls, you should first see which calls are happening that week and then Slack message the event organizer to ask if you can join. Look at Sales or CE team members' calendars to find calls. If the calendar invite was created by the prospect or customer, ask the Sales or CE team member before you add yourself.
```

> If you're working on a project and want to find calls about specific subjects, you can also post in the #CE and #Sales slack channels that you're looking for those calls.  

Reviewer: ```suggestion
If you're working on a project and want to find calls about specific subjects, you can also post in the #CE and #Sales Slack channels that you're looking for those calls.  
```

You should feel comfortable asking to join any call – the call organizer will let you know if you can't join for any reason. If you're "joining a call," they'll expect that you will silently observe. 

## How to join calls to speak

> If you want to speak directly with customers or prospects, you should first post in the #CE and #Sales channels what you'd like to talk about and with what (types of) customers or prospects. Then you'll work with the CE and Sales team members directly to set up new calls or join existing ones. 

Reviewer 2: @Joelkw For "How to join calls to speak", we need to require more than "...what you'd like to talk about and with what (types of) customers or prospects." At a minimum/as a start, there should be a predetermined list of questions looking to be asked (shared with the Sales/CE team), profile of account interested in (as specific as possible - an account with thousands of engineers? with a lot of legacy code? etc.), purpose of call (disprove a hypothesis, feedback gathering for a potential new feature, etc.).

Reviewer 2: Thanks for adding, this is a great start for a much needed update!

Author: Agree! I've added a bit more detail but there's a lot of process around joining calls for user feedback worth documenting (the whole product team is working on generally! https://about.sourcegraph.com/handbook/product/goals#increase-customer-funnel-of-feedback). I'll plan to come back and add more documentation around joining customer calls for feedback once I've gone through the process a few times myself.
